pkg/cmd/pulumi/stack/secrets.go

```diff
@@ -107,7 +108,7 @@ func createSecretsManagerForNewStack(
 	if err != nil {
 		ps = &workspace.ProjectStack{}
 	} else {
-		ps, err = loadProjectStackByReference(project, stackRef)
+		ps, err = loadProjectStackByReference(project, stackRef, nil)
```

Comment:
I decided to not pass it through here, since we're creating a new stack. We could pass it through here, but it does result in a lot more calls needed to be updated.